lib/openid/consumer/idres.rb

```diff
@@ -203,6 +203,21 @@ def verify_return_to_base(msg_return_to)
 
       # Raises ProtocolError if the signature is bad
       def check_signature
+        # ----------------------------------------------------------------------
+        # The server url must be defined within the endpoint instance for the
+        # OpenID2 namespace in order for the signature check to complete
+        # successfully.
+        #
+        # This fix corrects issue #125 - Unable to complete OpenID login
+        #                                with ruby-openid 2.9.0/2.9.1
```

Comment:
Probably we can ✂️  this out from here?

Reply:
That is fine to remove those two comment lines during the merge.